functions_lab.js

```diff
@@ -0,0 +1,283 @@
+// # Function Exercises
```

Comment:
Overall good demonstration of functions and scope concepts. No major errors or feedback but always remember to minimize side effects (i.e: declaring variables outside). Otherwise good job and keep up the good work!